component/listener/listener.go

```diff
@@ -65,12 +65,12 @@ type Read struct {
 	alias      string
 }
 
-func NewRead(batcher b.MqBatcher[b.MessageArguments], conf conf.Connection, schemas conf.SchemasConf, book string, group string, alias string) (*Read, error) {
+func NewListener(batcher b.MqBatcher[b.MessageArguments], conf conf.Connection, schemas conf.SchemasConf, book string, group string, alias string) (*Listener, error) {
```

Comment:
note: if the package has the name `listener`, there is no need to add a listener suffix to the `New` method. `listener.New`, in my opinion, adds less noise than `listener.NewListener`